Package/Captcha/Captcha.php

```diff
@@ -410,6 +428,28 @@ public function setSize($size)
     }
 
     /**
+     * Set widget language code
+     * @param string $languageCode
+     * @return Captcha
+     * @see https://developers.google.com/recaptcha/docs/language
+     */
+    public function setLanguageCode($languageCode)
+    {
+        $this->languageCode = $languageCode;
+        return $this;
+    }
+    
+    /**
+     * Get widget language code
+     * @return string
+     * @see https://developers.google.com/recaptcha/docs/language
```

Comment:
@AlekseyKorzun Done. But some other methods (like: setTheme(), setSize(), setType()) have the @see above @return. Actually I wrote phpDoc of the setLanguageCode() according to them.